### PR TITLE
BugFix disco_elasticgroup get_existing_group subnetIds

### PR DESCRIPTION
**Pull request by cl3086**
Seems like the response returned for a group doesn't have `subnetId`, it has `subnetIds`. The error that we have been seeing: ```File "/Users/calin/Projects/asiaq/disco_aws_automation/disco_elastigroup.py", line 92, in get_existing_groups
    zone['subnetId'] for zone in group['compute']['availabilityZones']
  File "/Users/calin/Projects/asiaq/disco_aws_automation/disco_elastigroup.py", line 92, in <genexpr>
    zone['subnetId'] for zone in group['compute']['availabilityZones']
KeyError: 'subnetId'```

An example group output `group['compute']['availabilityZones']`: `[{'subnetIds': ['subnet-0dc1d4a5c2ab3cc3e', 'subnet-0e0180ce15000d5ee'], 'name': 'us-west-2a'}, {'subnetIds': ['subnet-0d02ad259c7d1fe94', 'subnet-0796c13bae305148f'], 'name': 'us-west-2c'}, {'subnetIds': ['subnet-015132fcb1a50cc63', 'subnet-0a053166e0434cf53'], 'name': 'us-west-2b'}]`.

So this loops through each list and creates a comma-separated string of subnet IDs.

`','.join(subnet for subnets in test for subnet in subnets['subnetIds'])
'subnet-0dc1d4a5c2ab3cc3e,subnet-0e0180ce15000d5ee,subnet-0d02ad259c7d1fe94,subnet-0796c13bae305148f,subnet-015132fcb1a50cc63,subnet-0a053166e0434cf53'`

### CODE REVIEW

File: disco_aws_automation/disco_elastigroup.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def get_existing_groups(self, hostclass=None, group_name=None):
                 'launch_config_name': None,
                 'termination_policies': [],
                 'vpc_zone_identifier': ','.join(
-                    zone['subnetId'] for zone in group['compute']['availabilityZones']
+                    subnet for subnets in group['compute']['availabilityZones'] for subnet in subnets['subnetIds']
```
Comment (jconstance-amplify) [inline disco_aws_automation/disco_elastigroup.py:91]: Could improve the naming of the variables:
```suggestion
                    subnet for zone in group['compute']['availabilityZones'] for subnet in zone['subnetIds']
```

**Review by jconstance-amplify — DISMISSED** [review]

File: disco_aws_automation/disco_elastigroup.py (hunk 1 above)
Comment (jconstance-amplify) [reply disco_aws_automation/disco_elastigroup.py:91]: Also line is too long.

**Review by jconstance-amplify — APPROVED** [review]

**Comment by aoskotsky-amplify** [comment]
How did this ever work?

**Comment by jconstance-amplify** [comment]
Probably another fun breaking change from Spotinst, just like how they broke their logging API last year.

**Pull request merged**